src/services/chain/messages/cosmos/msgSend.ts

```diff
@@ -72,19 +70,7 @@ export class CosmosSendMessage extends CosmosMessageBase<SendToolParams> {
    * @returns Transaction object ready for signing
    */
   async buildTransaction(params: SendToolParams): Promise<string> {
-    // const { fromAddress, toAddress, amount, denom } = params;
-    // {
-    //   typeUrl: MessageTypeUrl.COSMOS_MSG_SEND,
-    //   value: {
-    //     fromAddress,
-    //     toAddress,
-    //     amount: {
-    //       amount,
-    //       denom,
-    //     },
-    //   },
-    // };
-
+    console.log(params);
```

Comment:
Satisfy `npm run check` - this will likely get deprecated anyways (unless we want to support sending cosmos assets between kava wallets)